hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -736,6 +736,7 @@ void Vgen::emit(const decqmlock& i) {
   if (RuntimeOption::EvalJitArmLse) {
     a->Mov(rVixlScratch0, -1);
     a->stadd(rVixlScratch0, adr);
+    a->Msr(NZCV, vixl::xzr);
```

Comment:
The NZCV flags are set by the ARM sub(s) instruction based on the SetFlags parameter on line 744.